docs/content/docs/install/manual_demo/_index.md

```diff
@@ -244,9 +244,63 @@ At this point, applications do not have access to each other because no access c
 
 Apply the [SMI Traffic Target][1] and [SMI Traffic Specs][2] resources to define access control and routing policies for the applications to communicate:
 
+ Deploy SMI TrafficTarget
 ```bash
-# Deploy SMI TrafficTarget and HTTPRouteGroup policy
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.8/docs/example/manifests/access/traffic-access-v1.yaml
+kubectl apply -f - <<EOF
```

Comment:
Could we combine these into a single `kubectl` command to save a copy-paste cycle?

Reply:
@nojnhuh I'd like to keep these separate steps. My goal is to leverage the the kinetics of having to copy/paste twice is a clear indicator these are 2 separate entities and YET linked to each other.

I'll probably add one extra line between the two explaining how the two are related - PR coming!